### ISSUE TITLE
multiple plots with optim.logger

### ISSUE THREAD

**Issue by YitzhakSp** [torch/optim#125]
is there a way to have multiple plots in one figure? one plot works fine, but when I do something like
`logger:add{['training loss'] = loss1 };
logger:add{['test loss'] = loss2} `
it gives an error. so I define two loggers but the disadvantage is, that they produce two figures instead of one.

**Comment by ajdroid** [comment]
@YitzhakSp , why did you close this?

**Comment by YitzhakSp** [comment]
because the answer is found in the documentation:

https://github.com/torch/optim/blob/master/doc/logger.md
